draft-ietf-quic-load-balancers.md

```diff
@@ -482,6 +482,41 @@ gets large relative to its share of the codepoint space, so that these
 allocations time out and are freed for reuse by servers that have recently
 joined the pool.
 
+## CID format
+
+All connection IDs use the following format:
+
+~~~
+QUIC-LB Connection ID {
+    First Octet (8),
+    Server ID (..),
+    Nonce (..),
+    For Server Use (..),
+}
+~~~
+{: #plaintext-cid-format title="CID Format"}
+
+Each configuration specifies the length of the Server ID and Nonce fields, with
+limits defined for each algorithm.
+
+The server ID is assigned to each server in accordance with {{sid-allocation}}.
```

Comment:
nit: Should `server ID` and `nonce` below have the same capitalization as they do above?